TrackerMC/src/StrawDigisFromStrawGasSteps_module.cc

```diff
@@ -1258,13 +1258,20 @@ namespace mu2e {
       float dw = delta.Dot(sdir);
       XYZVectorF cperp = delta - dw*sdir; // just perp part
       float phi = atan2(cperp.Dot(pdir),cperp.Dot(zdir)); // angle around wire WRT z axis in range -pi,pi
-      float rho = min(sqrt(cperp.mag2()),(float)_rstraw); // truncate!
+      float rho = sqrt(cperp.mag2());
```

Comment:
I am confused how cpos could ever be outside the straw, since it comes from the G4 energy deposits.

Reply:
The position is randomly adjusted here https://github.com/Mu2e/Offline/blob/030e4c8178023f2259a37d23c4a42d84d6bc763c/TrackerMC/src/StrawDigisFromStrawGasSteps_module.cc#L934